apps/dashboard/app/views/scripts/show.html.erb

```diff
@@ -2,4 +2,8 @@
 <h1><%= @script.title %></h1>
 
 
-<%= link_to 'Back', project_path(params[:project_id]), class: 'btn btn-default mt-3',  title: 'Return to projects page' %>
\ No newline at end of file
+<%= link_to 'Back', project_path(params[:project_id]), class: 'btn btn-default mt-3',  title: 'Return to projects page' %>
+
+<%= form_tag project_script_path(@script.project_dir, @script.id), method: :put do %>
+<%= select_tag 'smart_attribute', options_for_select(@smart_attributes) %>
+<% end %>
```

Comment:
Let's not worry about the form just yet, I'd say start with this:

```
<%= @script.smart_attributes.inspect %>
```
for now. At some point we'll loop through and build the form that will `#submit`. But for right now, I'd say just make sure that the internals work before we tackle the externals.